content/guides/public-participation/_index.md

```diff
@@ -7,7 +7,9 @@ guide: public-participation
 aliases:
 
 ---
+
 {{< guide-toc >}}
```

Comment:
Should we remove this now that it's deleted?

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/91551e3a-ebb9-4b15-840e-eced16822e7b)